### PR TITLE
add xml and jpeg test

### PR DESCRIPTION
**Pull request by 09041985**
@dmitrytokarev Take a look please

### CODE REVIEW

File: test_httpbin_ip.py [hunk 1]
```diff
@@ -53,6 +53,20 @@ def test_basic_auth_2(base_url, user, password):
     assert r.json() == {'authenticated': True, 'user': user}
 
 
+
+def test_xml(base_url):
+    r = get(base_url + 'xml')
+    assert r.status_code == 200
+    assert r.headers.get('Content-Type') == 'application/xml'
+    assert r.headers.get('Content-Length') == '522'
```
Comment (dmitrytokarev) [inline test_httpbin_ip.py]: missing checks for r.reason, and some body validation (r.text)

File: test_httpbin_ip.py [hunk 2]
```diff
@@ -53,6 +53,20 @@ def test_basic_auth_2(base_url, user, password):
...
+
+
+def test_jpeg(base_url):
+    r = get(base_url + 'image/jpeg')
+    assert r.status_code == 200
+    assert r.headers.get('Content-Type') == 'image/jpeg'
+    assert r.headers.get('Content-Length') < '4000'
```
Comment (dmitrytokarev) [inline test_httpbin_ip.py]: the response should have a binary body - we will look into that at the class.

Comment (dmitrytokarev) [reply test_httpbin_ip.py]: also need to verify r.reason, and elapsed

**Comment by 09041985** [comment]
@dmitrytokarev Take a look, added tests for binary data(images), svg is written in a separate test, the format of the Content-Type was different (svg+xml)

**Comment by dmitrytokarev** [comment]
@09041985 you still need to remove `downloaded.jpeg` file

File: test_httpbin_ip.py [hunk 3]
```diff
@@ -153,3 +161,37 @@ def test_redirect_to_absolute(config):
...
+    with open('open_temp_file', 'wb') as file:
+        file.write(r.content)
+    assert filecmp.cmp('open_temp_file', data_file_path, shallow=False)
+    assert difflib.ndiff('open_temp_file', data_file_path)
+
+
+
+
+
+
```
Comment (dmitrytokarev) [inline test_httpbin_ip.py]: please remove extra lines. should have only 1 empty line at the end of the file

File: test_httpbin_ip.py [hunk 4]
```diff
@@ -153,3 +161,37 @@ def test_redirect_to_absolute(config):
...
+    filename = img_name
+    path_to_current_file = os.path.realpath(__file__)
+    current_directory = os.path.split(path_to_current_file)[0]
+    data_file_path = os.path.join(current_directory, "images", filename)
+    file_binar_data = open(data_file_path, 'rb').read()
+
+    assert r.content == file_binar_data
+
+    with open('open_temp_file', 'wb') as file:
+        file.write(r.content)
```
Comment (dmitrytokarev) [inline test_httpbin_ip.py]: still need to rename `open_temp_file` to `temp_file`.
open_temp_file is a name for action like a method, but here it should be just a noun to name the temporary file (e.g. temp_file, tempfile, dummy_file, etc.)
Additionally don't use built-in keywords as variable names. `file` is a reserved word in Python to denote a type of a file object. I usually just use `f`, you can use `myfile` or `tempfile`.

File: test_httpbin_ip.py [hunk 5]
```diff
@@ -153,3 +161,37 @@ def test_redirect_to_absolute(config):
...
+    assert r.reason == 'OK'
+    assert r.headers.get('Content-Type') == 'image/' + content_type
+
+    filename = img_name
+    path_to_current_file = os.path.realpath(__file__)
+    current_directory = os.path.split(path_to_current_file)[0]
+    data_file_path = os.path.join(current_directory, "images", filename)
+    file_binar_data = open(data_file_path, 'rb').read()
+
+    assert r.content == file_binar_data
```
Comment (dmitrytokarev) [inline test_httpbin_ip.py]: typo: should be file_binary_data

File: test_httpbin_ip.py [hunk 6]
```diff
@@ -153,3 +161,31 @@ def test_redirect_to_absolute(config):
...
+    filename = img_name
+    path_to_current_file = os.path.realpath(__file__)
+    current_directory = os.path.split(path_to_current_file)[0]
+    data_file_path = os.path.join(current_directory, "images", filename)
+    file_binary_data = open(data_file_path, 'rb').read()
+
+    assert r.content == file_binary_data
+
+    with open('tempfile', 'wb') as file:
+        file.write(r.content)
```
Comment (dmitrytokarev) [inline test_httpbin_ip.py]: As already mentioned in previous comments:
```
Additionally don't use built-in keywords as variable names. file is a reserved word in Python to denote a type of a file object. I usually just use f, you can use myfile or tempfile.
```

Comment (dmitrytokarev) [reply test_httpbin_ip.py]: @09041985 please fix this

File: test_httpbin_ip.py (hunk 1 above)
Comment (dmitrytokarev) [reply test_httpbin_ip.py]: @09041985 please add missing checks

**Comment by 09041985** [comment]
@dmitrytokarev Take a look, renamed variable.

**Pull request merged**